package.json

```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "dev": "nodemon --config ./nodemon.json --exec 'yarn build && yarn start'",
     "lint": "eslint --ext .ts .",
```

Comment:
We will need this flag until this issue is closed:
https://github.com/nodejs/node/issues/51154